draft-ietf-spice-sd-cwt.md

```diff
@@ -684,6 +737,19 @@ Security considerations from COSE {(RFC9052)} and CWT {{RFC8392}} apply to this
 Each salt used to protect disclosed claims MUST be generated independently from the salts of other claims. The salts MUST be generated from a source of entropy that is acceptable to the issuer.
 Poor choice of salts can lead to brute force attacks that can reveal redacted claims.
 
+## Binding the KBT and the CWT
+
+The issuer claim in the SD-CWT is self-asserted by the issuer.
+The subject claim of the SD-CWT is the Holder.
```

Comment:
```suggestion
Because confirmation is mandatory, the subject claim of an SD-CWT, when present, is always related directly to the confirmation claim.

There might be many subject claims and many confirmation keys that identify the same entity or that are controlled by the same entity, while the identifiers and keys are distinct values.

Reusing an identifier or key enables traceability, but MUST be evaluated in terms of the confidential and privacy constraints associated with the credential type.
```